ultimatejobweb/templates/available_jobs.html

```diff
@@ -61,19 +66,28 @@ <h1 class="display-3">UltimateJob</h1>
         </div>
 
    <!--tables-->
-   <div class="container">
+	<div class="container">
     <table class="table table-hover table-stripped">
         <thead class="thead-dark">
             <tr>
                 <th>Company name</th>
                 <th>Position offered</th>
-                <th>Upload date</th>
-                <th>Location</th>
                 <th>Link to job application</th>
             </tr>
         </thead>
         <tbody>
-            <tr>
+		   <h1> Available jobs  </h1>
+		   <div>
+				{% for item in job_posts %}
```

Comment:
The parameter passed in here from views is Jobs, so this probably needs to be:
for item in Jobs.
Jobs should be lowercase, as it is a variable that contains the class, so this should be changed in views.py - available_jobs function.